shared/common-adapters/clickable-box.desktop.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react'
+import FlatButton from 'material-ui/FlatButton'
+import Box from './box'
+
+import type {Props} from './clickable-box'
+
+export default function ClickableBox (props: Props & {children: any}) {
+  return (
+    <FlatButton {...props} style={{textAlign: 'left', height: undefined, width: '100%', display: 'flex', flexDirection: 'column', alignItems: 'stretch'}}>
```

Comment:
:sheep: A thought: you could do `{style, ...otherProps} = props` here so that `style` doesn't get included in the `{...props}` and immediately overridden.